events/wc_undead_events.txt

```diff
@@ -1285,6 +1293,102 @@ character_event = {
 			factor = 25
 		}
 	}
+	
+	# Wickerman
+	option = {
+		trigger = {
+			event_target:target_undead = { NOT = {  trait = creature_wickerman } }
+			
+			NOT = { event_target:target_undead = { is_paladin_class_trigger = yes } }
+			event_target:target_necromancer = {
+				OR = {
+					has_7_8_magic_class_or_higher_trigger = yes
+					AND = {
+						religion = throsic
+					}
+				}
+			}
+		}
+		name = EVTOPTG_WCUND_300
+
+		event_target:target_undead = {
+			show_scope_change = no
+			
+			ROOT = {
+				show_scope_change = no
+				
+				complete_quest_turn_into_undead_prev_effect = yes
+			}
+		}
+		event_target:target_undead = {
+			set_character_flag = transition_into_wickerman_flag
+			tooltip = {
+				remove_racial_traits_effect = yes
+				add_trait = creature_wickerman
+				add_trait_undead_effect = yes
+				if = {
+					limit = { event_target:target_undead = { NOT = { character = ROOT } } }
+					opinion = {
+						modifier = opinion_loyal_servant
+						who = ROOT
+					}
+				}
+			}
+		}
+		hidden_effect = { event_target:target_undead = { character_event = { id = WCPOD.5013 } } }
+
+		ai_chance = {
+			factor = 25
+		}
+	}
+	
+	# Wicker behemoth
```

Comment:
I thought it's some kind of abomination (big monstrous undead). What is it doing in the event chain to raise small undead?

Reply:
Behemoths =/= brutes, they are the 'middle' ground